src/migrate/arcgis/index.md

```diff
@@ -0,0 +1,96 @@
+
+This tool helps users transition smoothly by preserving complex symbology, layouts, and project structure. It simplifies the shift from Esri tools to the open-source QGIS environment, reducing the time and effort needed to replicate existing workflows. 
+
+SLYR tool offers Community and Licensed version:
+- Community version is open-sourced on <GitHubRepo id="north-road/slyr" desc="GitHub" /> and can be freely installed via QGIS Plugin Manager to extract, parse, and convert Esri `.lyr` and `.style` files to QGIS-compatible formats. 
+- Licensed version is closed-sourced and has features such as converting `MXD/MXT` project files to `QGS` or converting various data formats to GeoPackage. 
+
+Full comparison of the version can be found on North Road's [SLYR project page](https://north-road.com/slyr/). 
+
+![SLYR logo](./slyr.png "SLYR logo")
+
+
+### Domains 
+
+ArcGIS Pro [domains](https://pro.arcgis.com/en/pro-app/latest/help/data/geodatabases/overview/create-modify-and-delete-domains.htm) are used to restrict the valid values allowed in an attribute field. Convert them manually to QGIS drop-down widgets - likely [Value Relation](../../layer/form-widgets) - could be time consuming. You can export the domains to CSV to be imported and set value relation widgets, along with it's custom expression visibility to mimic cascading forms. We again recommend using [SLYR](#slyr) to facilitate the conversion of domains to QGIS.
+
+### Data Formats
+
+ArcGIS Pro [supports GeoPackages](https://pro.arcgis.com/en/pro-app/latest/help/data/databases/work-with-sqlite-databases-in-arcgis-pro.htm), which are the primary and recommended format for your survey layer in <MainPlatformName />. Both ArcGIS and QGIS can also handle Shapefile, GeoTIFF, GeoJSON, WMS, WFS, and PostGIS layers. However, we do not recommend using [Shapefile](http://switchfromshapefile.org) in <MainPlatformName />, as we do not support collaborative use for this format.
+
+There are multiple options for data conversion. We recommend using [SLYR](#slyr) processing algorithms to convert project data to GeoPackage (GPKG). You can also use QGIS *Convert format* or *Package layers* processing algorithms. Alternatively, export your data to [formats supported by <MainPlatformName />](../../gis/supported_formats.md) in ArcGIS Pro, if you feel more comfortable doing it there. Data conversion can be done also in the console using the `ogr2ogr` command line tool.
+
+## Hybrid setup: using Mergin Maps with ArcGIS
+
+It is possible to use ArcGIS and <MainPlatformName /> in a hybrid setup. If you store your data in PostgreSQL/PostGIS database, you can use our [DB Sync](../../dev/dbsync/) tool to synchronise project data from PostgreSQL to <MainPlatformName /> (i.e. storing data in GeoPackages on <MainPlatformName /> server). You still need to have a QGIS project to style your GeoPackage data and set up the project (see the [Migrate section](#migrate-from-arcgis-to-qgis)).
```

Comment:
The same should work for Geodatabases 👍

Reply:
It will not work again because Mergin Maps (geodiff) does not support geodatabases for collaborative work.